### PR TITLE
3528 fieldset border

### LINKED ISSUE #3528: /Kubernetes/contact-us - Fix contact us - DEV
please fix on all contact us pages

screencapture-beta-staging-ubuntu-kubernetes-contact-us-2018-07-03-12_06_14

---
*Reported from: https://beta.staging.ubuntu.com/kubernetes/contact-us?product=kubernetes-explorer*

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Change colour of fieldset border

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/kubernetes/contact-us>
- See that grey box on form has a border

## Issue / Card

Fixes #3528

### CODE REVIEW

**Comment by anasereijo** [comment]
@steverydz 
- I think you forgot to make the CAPTCHA padding as requested
- the border looks good but should be just on the outside and at the moment there's an "internal" border too:
screen shot 2018-07-03 at 16 53 44

**Comment by steverydz** [comment]
@anasereijo Done

File: static/sass/styles.scss [hunk 1]
```diff
@@ -323,6 +323,19 @@ p + pre {
...
 fieldset {
-  border: 1px solid $color-light;
-  border-radius: $sp-xx-small;
+  border: 1px solid $color-mid-light;
+  border-bottom-width: 0;
+  border-radius: $sp-xx-small $sp-xx-small 0 0;
+}
+
+fieldset + fieldset {
+  border-top-width: 0 !important;
```
Comment (Caleb-Ellis) [inline static/sass/styles.scss]: @steverydz Do we need any of these `!important`s?

Comment (steverydz) [reply static/sass/styles.scss]: Yes, it wasn't working without them sadly :(

**Pull request merged**